binr/r2pm/d/esilburner

```diff
@@ -0,0 +1,18 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/mrmacete/r2scripts"
+R2PM_DESC "[r2-r2pipe-python] Burn asm.emu results in real comments for all detected functions (MIPS oriented)"
+
+R2_RCD="${R2PM_PLUGDIR}/../radare2rc.d"
+
+R2PM_INSTALL() {
+	mkdir -p "${R2_RCD}"
+	cd esilburner
+	echo '$esilburner=#!pipe python '"'`pwd`'"'/esilburner.py' > "${R2_RCD}/esilburner"
```

Comment:
Theres a global rc file but not a global dir for config, there was an issue to discuss this already. Not sure if its still open. And maybe this directory must be in etc instead of share

> On 31 May 2016, at 19:16, @bezjaje <notifications@github.com> wrote:
> 
> In binr/r2pm/d/esilburner:
> 
> > @@ -0,0 +1,18 @@
> > +R2PM_BEGIN
> > +
> > +R2PM_GIT "https://github.com/mrmacete/r2scripts"
> > +R2PM_DESC "[r2-r2pipe-python] Burn asm.emu results in real comments for all detected functions (MIPS oriented)"
> > +
> > +R2_RCD="${R2PM_PLUGDIR}/../radare2rc.d"
> > +
> > +R2PM_INSTALL() {
> > +	mkdir -p "${R2_RCD}"
> > +	cd esilburner
> > +	echo '$esilburner=#!pipe python '"'`pwd`'"'/esilburner.py' > "${R2_RCD}/esilburner"
> isn't there a "global" radare2rc.d ?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or mute the thread.
>